include.js

```diff
@@ -292,6 +300,13 @@ let cards = [
     author: 'Simar',
     githubLink: 'https://github.com/SimardeepSingh-zsh'
   },
+  {
+    artName: 'solarsystem',
+    pageLink: './Art/Joy/solarsystem.html',
+    imageLink: './Art/Joy/solarsystem.gif',
```

Comment:
Same here, your image is not in the `Joy` folder, but in the `solarsystem` folder.